README.jp.md

```diff
@@ -0,0 +1,805 @@
+# Ctrl+Cで録画を中断する
+```
+
+"スキップされたフレーム"はリアルタイムで表示されなくても録画されます
+(パフォーマンス上の理由)。
+
+フレームはデバイス上で _タイムスタンプされる_ ため [パケット遅延のバリエーション] は
+録画されたファイルに影響を与えません。
+
+[パケット遅延のバリエーション]: https://en.wikipedia.org/wiki/Packet_delay_variation
+
+
+### 接続
+
+#### ワイヤレス
+
+_Scrcpy_ はデバイスとの通信に`adb`を使用します。そして`adb`はTCP/IPを介し
+デバイスに[接続]することができます:
+
+1. あなたのコンピュータと同じWi-Fiに接続します。
+2. あなたのIPアドレスを取得します。設定 → 端末情報 → ステータス情報、
+   もしくは、このコマンドを実行します:
+
+    ```bash
+    adb shell ip route | awk '{print $9}'
+    ```
+
+3. あなたのデバイスでTCP/IPを介したadbを有効にします: `adb tcpip 5555`
+4. あなたのデバイスの接続を外します。
+5. あなたのデバイスに接続します:
+ `adb connect DEVICE_IP:5555` _(`DEVICE_IP`は置き換える)_
+6. 通常通り`scrcpy`を実行します。
+
+この方法はビットレートと定義を減らすのにおそらく有用です:
+
+```bash
+scrcpy --bit-rate 2M --max-size 800
+scrcpy -b2M -m800  # short version
+```
+
+[接続]: https://developer.android.com/studio/command-line/adb.html#wireless
+
+
+#### マルチデバイス
+
+もし`adb devices`でいくつかのデバイスがリストされる場合、 _シリアルナンバー_ を
+指定する必要があります:
+
+```bash
+scrcpy --serial 0123456789abcdef
+scrcpy -s 0123456789abcdef  # short version
+```
+
+デバイスがTCP/IPを介して接続されている場合:
+
+```bash
+scrcpy --serial 192.168.0.1:5555
+scrcpy -s 192.168.0.1:5555  # short version
+```
+
+複数のデバイスに対して、複数の _scrcpy_ インスタンスを開始することができます。
+
+#### デバイス接続での自動起動
+
+[AutoAdb]を使用可能です:
+
+```bash
+autoadb scrcpy -s '{}'
+```
+
+[AutoAdb]: https://github.com/rom1v/autoadb
+
+#### SSHトンネル
+
+リモートデバイスに接続するため、ローカル`adb`クライアントからリモート`adb`サーバーへ
+接続することが可能です(同じバージョンの _adb_ プロトコルを使用している場合):
+
+```bash
+adb kill-server    # kill the local adb server on 5037
+ssh -CN -L5037:localhost:5037 -R27183:localhost:27183 your_remote_computer
+# keep this open
+```
+
+他の端末から:
+
+```bash
+scrcpy
+```
+
+リモートポート転送の有効化を回避するためには、代わりに転送接続を
+強制することができます(`-R`の代わりに`-L`を使用することに注意):
+
+```bash
+adb kill-server    # kill the local adb server on 5037
+ssh -CN -L5037:localhost:5037 -L27183:localhost:27183 your_remote_computer
+# keep this open
+```
+
+他の端末から:
+
+```bash
+scrcpy --force-adb-forward
+```
+
+
+ワイヤレス接続と同様に、クオリティを下げると便利な場合があります:
+
+```
+scrcpy -b2M -m800 --max-fps 15
+```
+
+### ウィンドウ構成
+
+#### タイトル
+
+ウィンドウのタイトルはデバイスモデルが初期値です。これは変更できます:
+
+```bash
+scrcpy --window-title 'My device'
+```
+
+#### 位置とサイズ
+
+ウィンドウの位置とサイズの初期値を指定できます:
+
+```bash
+scrcpy --window-x 100 --window-y 100 --window-width 800 --window-height 600
+```
+
+#### ボーダーレス
+
+ウィンドウの装飾を無効化するには:
+
+```bash
+scrcpy --window-borderless
+```
+
+#### 常に画面のトップ
+
+scrcpyの画面を常にトップにするには:
+
+```bash
+scrcpy --always-on-top
+```
+
+#### フルスクリーン
+
+アプリケーションを直接フルスクリーンで開始できます:
+
+```bash
+scrcpy --fullscreen
+scrcpy -f  # short version
+```
+
+フルスクリーンは、次のコマンドで動的に切り替えることができます <kbd>MOD</kbd>+<kbd>f</kbd>
+
+
+#### 回転
+
+ウィンドウは回転することができます:
+
+```bash
+scrcpy --rotation 1
+```
+
+設定可能な値:
+ - `0`: 回転なし
+ - `1`: 90° 反時計回り
+ - `2`: 180°
+ - `3`: 90° 時計回り
+
+回転は次のコマンドで動的に変更することができます。 <kbd>MOD</kbd>+<kbd>←</kbd>
+_(左)_ 、 <kbd>MOD</kbd>+<kbd>→</kbd> _(右)_
+
+_scrcpy_ は3つの回転を管理することに注意:
+ - <kbd>MOD</kbd>+<kbd>r</kbd>はデバイスに縦向きと横向きの切り替えを要求する
+   (現在実行中のアプリで要求している向きをサポートしている場合、拒否する場合がある)
```

Comment:
@rom1v The original text is

> the current running app may refuse, if it does support the requested orientation

Shouldn't it be "if it **doesn't** support"

Reply:
Thanks :+1: 1fb79575250460e159161bbe5ca55335409ffec5